### ISSUE TITLE
Nested object query does not work

### ISSUE THREAD

**Issue by Ehesp** [balderdashy/sails-disk#47]
Just had an object in my local disk:

```
{
  id: '1234',
  nested: {
    content: {
      userId: 'foobar'
    }
  }
}
```

Running the below returns no results, however when switching to Mongo, the results are returned:

```
User.find( { 'nested.content.userId': 'foobar' }).then(....
```

Running `"sails-disk": "^0.10.8"`

Thanks

**Comment by sailsbot** [comment]
Thanks for posting, @Ehesp.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
